packages/replay/src/replay.ts

```diff
@@ -1139,6 +1139,7 @@ export class ReplayContainer implements ReplayContainerInterface {
         `[Replay] Session duration (${Math.floor(duration / 1000)}s) is too ${
           tooShort ? 'short' : 'long'
         }, not sending replay.`,
+        this._options._experiments.traceInternals,
```

Comment:
Also noticed this, missed this in the refactor.

Reply:
I might have messed this up when rebasing #8716, sorry!

Reply:
no big deal either way, it's just internal logging ;)